resources/js/features/events/components/TopEventPlayers/TopEventPlayers.tsx

```diff
@@ -0,0 +1,98 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseTable,
+  BaseTableBody,
+  BaseTableCell,
+  BaseTableHeader,
+  BaseTableRow,
+} from '@/common/components/+vendor/BaseTable';
+
+import type { TopPlayersListKind } from './top-players-list-kind.model';
+import { TopEventPlayersRow } from './TopEventPlayersRow';
+
+interface TopEventParticipantsProps {
+  event: App.Platform.Data.Event;
+  numMasters: number;
+  players: App.Platform.Data.GameTopAchiever[];
+}
+
+export const TopEventPlayers: FC<TopEventParticipantsProps> = ({ event, numMasters, players }) => {
+  const { t } = useTranslation();
+
+  if (!players.length) {
+    return null;
+  }
+
+  const areAllAchievementsOnePoint = !!event.eventAchievements?.every(
+    (ea) => ea.achievement?.points && ea.achievement.points === 1,
+  );
+
+  const listKind = getListKind(areAllAchievementsOnePoint, numMasters);
+
+  return (
+    <div data-testid="top-players">
+      <h2 className="mb-0 border-0 text-lg font-semibold">
+        {listKind === 'latest-masters' ? t('Latest Masters') : null}
+        {listKind === 'most-achievements-earned' ? t('Most Achievements Earned') : null}
+        {listKind === 'most-points-earned' ? t('Most Points Earned') : null}
+      </h2>
+
+      <div className="flex flex-col gap-2 rounded-lg bg-embed p-2">
+        <BaseTable className="table-highlight overflow-hidden rounded-lg outline outline-1 outline-neutral-800">
+          <BaseTableHeader className="border-neutral-800">
+            <BaseTableRow className="do-not-highlight text-menu-link">
+              <BaseTableCell className="text-right">{'#'}</BaseTableCell>
```

Comment:
I realize this matches existing behavior, and I'm assuming it's using the same logic/cache to retrieve the data, so that's understandable.

But should this be updated to share ranks for ties like is done on the "See More" page?
![image](https://github.com/user-attachments/assets/e1880d6f-b647-4858-b68c-da6e2637037d)
![image](https://github.com/user-attachments/assets/f9d58d6f-9bb7-43da-b996-5f52ea9f8ac1)

Reply:
Updated in latest so ties are properly handled in the UI.